src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -52,17 +52,40 @@ import scala.collection.immutable.ListMap
     |
     |## Available Filters
     |
+    |### A-tailing Artifact Filter
+    |
+    |The A-tailing artifact filter attempts to measure the probability that a SNV is the product of
+    |errors in the template generated during the A-base addition steps that are common to many Illumina
+    |library preparation protocols.  The artifacts occur if/when a recessed 3' end is incorrectly filled
+    |in with one or more adenines during A-base addition. This presents specifically as errors to T at
+    |the beginning of reads (and in very short templates, as matching errors to A at the ends of reads).
+    |
+    |The filter adds the `INFO` field `ATP` to SNVs with an A or T alternate allele. This field records
+    |the p-value for rejecting the possibility that the variant is an A-tailing artifact. If a threshold
+    |p-value is specified, then the `FILTER` tag `ATailingArtifact` will be applied to variants with
+    |p-values below the threshold.
```

Comment:
Just below (`<`), or below and equal to (`<=`)?

Reply:
```suggestion
    |p-values less than or equal to the threshold.
```